tests/test_nn/test_node_piece.py

```diff
@@ -103,3 +110,19 @@ class TokenizerMetaTestCase(unittest_templates.MetaTestCase[pykeen.nn.node_piece
 
     base_cls = pykeen.nn.node_piece.Tokenizer
     base_test = cases.TokenizerTestCase
+
+
+def test_page_rank():
+    """Test for page-rank code."""
+    n = 10
+    edge_index = numpy.stack(
+        [
+            numpy.arange(n),
+            (numpy.arange(n) + 1) % n,
+        ],
+    )
+    result = pykeen.nn.node_piece.page_rank(
+        edge_index=edge_index,
+        epsilon=1.0e-08,
+    )
+    numpy.testing.assert_allclose(result.sum(), 10.0, rtol=1e-6)
```

Comment:
@migalkin shouldn't the entries sum to one?

Reply:
Is it 10 because that's the value of `1.0 * n`? If so, it would be better to write the test to reflect that so the n could potentially be changed

Reply:
right, the pagerank values should indeed sum up to one, I double-checked that with `nx.pagerank()` of this line graph. It means the current implementation is incorrect because node states converge to ones 🤔